AdminClient/nginx.conf

```diff
@@ -0,0 +1,15 @@
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+    server {
+        listen 80;
```

Comment:
should we handle https in here or will it be done by a reverse-proxy?

Reply:
changed to 4200